pkgs/blast_repo/pubspec.yaml

```diff
@@ -9,6 +9,8 @@ environment:
 dependencies:
   args: ^2.3.1
   collection: ^1.17.0
+  firehose:
```

Comment:
I would avoid this path-relative dep if possible.

Reply:
I thought that `blast_repo` not being published means we can relax the principle of reducing the number of imports :) I am also fine with code duplication if that's the lesser evil.

Reply:
It's not wrong - avoiding publishing another package is more important. Perhaps add a line of docs here - `# Re-using the repo scanning logic from firehose` or similar.

Reply:
I am not sure I understand - does this mean you would rather want to copy the code or add the dep with a comment?